src/envoy/http/authn/authenticator_base.cc

```diff
@@ -25,6 +24,8 @@ namespace Http {
 namespace Istio {
 namespace AuthN {
 namespace {
+const std::string kJwtHeaderKey = "sec-istio-auth-userinfo";
```

Comment:
Add comment for this constant, also note that it would be removed once jwt filter support output parameters.

Reply:
Done.